framework/db/cubrid/QueryBuilder.php

```diff
@@ -29,6 +29,7 @@ class QueryBuilder extends \yii\db\QueryBuilder
         Schema::TYPE_INTEGER => 'int',
         Schema::TYPE_BIGINT => 'bigint',
         Schema::TYPE_FLOAT => 'float(7)',
+        Schema::TYPE_DOUBLE => 'double(7)',
```

Comment:
has to be `double(15)` as this is the default value. http://www.cubrid.org/manual/93/en/sql/datatype.html#double-double-precision